### PR TITLE
Remove pipreqsnb requirement from SDK

### PR DESCRIPTION
**Pull request by kenxu95**
Following's Joey's suggestion to remove inference in OSS for now: https://linear.app/aqueducthq/issue/ENG-1191/remove-pipreqsnb-for-requirements-discovery

### CODE REVIEW

File: .github/workflows/integration-tests.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: Integration Tests
 
 on:
   push:
-    branches: [ main ]
+    branches: [ ]
```
Comment (cw75) [inline .github/workflows/integration-tests.yml:5]: is this expected?

Comment (kenxu95) [reply .github/workflows/integration-tests.yml]: testing artifacts... fixed

**Comment by jegonzal** [comment]
Other than comment @cw75 listed it looks good to me.

**Review by jegonzal — APPROVED** [review]

**Comment by cw75** [comment]
@kenxu95 So now, if the user manually provided a `requirements.txt`, we will try to install any package inside this file to the server. Otherwise, we don't attempt to infer or install anything. Is my understanding correct?

**Comment by kenxu95** [comment]
@cw75 the server will try to install whats in the zipped requirements.txt, yes. Otherwise, nothing happens.

**Pull request merged**